AISKU/src/AISku.ts

```diff
@@ -259,8 +261,40 @@ export class AppInsightsSku implements IApplicationInsights {
                     _self.emptyQueue();
                     _self.pollInternalLogs();
                     _self.addHousekeepingBeforeUnload(_self);
+
+                    _addUnloadHook(onConfigChange(cfgHandler, () => {
+                        if (!_throttleMgr){
+                            _throttleMgr = new ThrottleMgr(_core);
+                        }
+                        //  && !_config.throttleMgrCfg.disabled
+                        if (_throttleMgr && !_throttleMgr.isReady() && _config.extensionConfig && _config.extensionConfig[_cfgSyncPlugin.identifier]) {
+                            // set ready state to true will automaatically trigger flush()
+                            _throttleMgr.onReadyState(true);
+                        }
+
+                        if (!_iKeySentMessage){ //should not send again
+                            if (!_config.connectionString && !isFeatureEnabled("disableIkeyDeprecationMessage", _config)) {
```

Comment:
Ok, been through the PR now, lets not call these `disableXXXX` as we have a double negative.

Lets rename to something shorter (because the string won't be compressible" and then change this so it reads

`if(.... && isFeatureEnabled("<theFeatureName>", _config)....`  so it's more of a positive rather than getting into mental gymnastics of "is not feature enabled (disable feature name)"

Reply:
Important!
And the `defaultConfigValues` (up at line 57) should set this version of the SDK defaults, which would have this as disabled by default for this release or at the very least it MUST define the feature names otherwise they will NOT be dynamic and all of the `onConfigChange` settings would never get called if we forced them on at the CDN level and the user did not supply any defaults.